AdyenComponents/Boleto/BoletoComponent.swift

```diff
@@ -0,0 +1,233 @@
+        style: FormComponentStyle = FormComponentStyle()
+    ) {
+        self.configuration = configuration
+        self.style = style
+        
+        socialSecurityNumberItem.isHidden.wrappedValue = false
+    }
+    
+    /// :nodoc:
+    private lazy var socialSecurityNumberItem: FormTextInputItem = {
+        let socialSecurityNumberItem = FormTextInputItem(style: style.textField)
+        socialSecurityNumberItem.title = localizedString(.boletoSocialSecurityNumber, localizationParameters)
+        socialSecurityNumberItem.placeholder = localizedString(.boletoSocialSecurityNumber, localizationParameters)
+        socialSecurityNumberItem.validator = LengthValidator(minimumLength: 1)
+        socialSecurityNumberItem.autocapitalizationType = .none
+        socialSecurityNumberItem.identifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: "socialSecurityNumberItem")
+        
+        return socialSecurityNumberItem
+    }()
+    
+    /// :nodoc:
+    internal lazy var sendCopyByEmailItem: FormSwitchItem = {
+        let sendCopyToEmailItem = FormSwitchItem(style: style.switch)
+        sendCopyToEmailItem.value = false
+        sendCopyToEmailItem.title = localizedString(.boletoSendCopyToEmail, localizationParameters)
+        sendCopyToEmailItem.identifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: "sendCopyToEmailItem")
+
+        return sendCopyToEmailItem
+    }()
+    
+    /// :nodoc:
+    internal lazy var billingAddressLabelItem: FormContainerItem = {
+        FormLabelItem(
+            text: configuration.shopperInfo.billingAddress?.formatted ?? "",
+            style: style.hintLabel,
+            identifier: ViewIdentifierBuilder.build(scopeInstance: self, postfix: "preFilledBillingAddress")
+        ).withPadding(padding: .init(top: 0.0, left: 0.0, bottom: 0.0, right: 0.0))
+    }()
+    
+    /// :nodoc:
+    private func headerFormItem(key: LocalizationKey) -> FormContainerItem {
+        FormLabelItem(
+            text: localizedString(key, localizationParameters),
+            style: style.sectionHeader,
+            identifier: ViewIdentifierBuilder.build(
+                scopeInstance: self,
+                postfix: localizedString(key, localizationParameters)
+            )
+        ).withPadding(padding: .init(top: 8, left: 0, bottom: 0, right: 0))
+    }
+    
+    /// :nodoc:
+    private lazy var formComponent: FormComponent = {
+        let component = FormComponent(
+            paymentMethod: paymentMethod,
+            configuration: AbstractPersonalInformationComponent.Configuration(fields: getFormFields()),
+            onCreatePaymentDetails: { [weak self] in self?.createPaymentDetails() },
+            style: style
+        )
+        setUpFields(for: component)
+        component.delegate = self
+        return component
+    }()
+    
+    /// :nodoc:
+    public lazy var viewController: UIViewController = formComponent.viewController
```

Comment:
Minor thing: can use computed property here

```swift
var viewController: UIViewController {
    formComponent.viewController
}
```

Reply:
yes, but why? `viewController` is never supposed to change.

Reply:
> Better memory and retain cycle management.

But, as I said, It is minor :)